io.edgehog.devicemanager.config.Telemetry.json

```diff
@@ -0,0 +1,14 @@
+{
+    "interface_name": "io.edgehog.devicemanager.config.Telemetry",
+    "version_major": 0,
+    "version_minor": 1,
+    "type": "properties",
+    "ownership": "server",
+    "mappings": [
+        {
+            "endpoint": "/request/%{interface_name}/periodS",
```

Comment:
I'd use `periodSeconds`. Units of measurement should not follow capitalization rules (otherwise reading `Ms` you wouldn't be able to tell aparte Megaseconds and milliseconds), but then again `/periods` is not clear at all, so I would stick with the full name